src/theme/provider.tsx

```diff
@@ -0,0 +1,9 @@
+import { defaultLightTheme } from "./defaultLight.css";
+
+type ThemeProviderProps = {
+  children?: React.ReactNode;
```

Comment:
IMO children should be undefined in this case

Reply:
React node is already undefined:
![CleanShot 2023-01-11 at 12 52 46@2x](https://user-images.githubusercontent.com/9116238/211799691-5bc4f317-dd7d-467a-9b5e-1f35950e89a3.jpg)

Reply:
ok so why not use like this 
```
children: React.ReactNode;
```